src/chi/javascript/components/drawer.js

```diff
@@ -153,7 +153,7 @@ class Drawer extends Component {
             }
             self._shown = true;
             self._drawerElem.dispatchEvent(
-              Util.createEvent(EVENTS.SHOW)
+              Util.createEvent(EVENTS.show)
```

Comment:
@SergioQCL `EVENTS` is exported and `EVENTS.show` `EVENTS.hide` are already used in other components. Changing it can be risky. I leave it lowercase 😉